### PR TITLE
[PM-14] Add workouts list

### PR DESCRIPTION
**Pull request by atulb98**

### CODE REVIEW

File: app/src/main/java/io/mochahub/powermeter/models/WorkoutSet.kt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package io.mochahub.powermeter.models
+
+data class WorkoutSet (
+    val weight: Double,
+    val reps: Int = 0
+) {
+    operator fun plus(increment: Int): WorkoutSet {
+        return this.copy(reps = if (reps + increment <= 100) reps + increment else reps)
+    }
```
Comment (zahin-mohammad) [inline app/src/main/java/io/mochahub/powermeter/models/WorkoutSet.kt]: What is this function doing?

Comment (atulb98) [reply app/src/main/java/io/mochahub/powermeter/models/WorkoutSet.kt]: implements the `+` operator for a set so that I can do: `set = set + 2`

**Review by zahin-mohammad — APPROVED** [review]
lgtm

File: app/src/main/java/io/mochahub/powermeter/models/WorkoutSet.kt [hunk 2]
```diff
@@ -26,6 +26,6 @@ data class WorkoutSet (
     }
 
     fun setReps(reps: Int): WorkoutSet {
-        return this.copy(reps = if (reps >= REP_MIN_LIMIT) reps else this.reps)
+        return this.copy(reps = if (reps in REP_MIN_LIMIT..REP_MAX_LIMIT) reps else this.reps)
```
Comment (zahin-mohammad) [inline app/src/main/java/io/mochahub/powermeter/models/WorkoutSet.kt:29]: Woa Fancy.

**Pull request merged**